scss/_nav.scss

```diff
@@ -26,6 +26,11 @@
     text-decoration: if($link-hover-decoration == underline, none, null);
   }
 
+  &:focus {
+    outline: 0;
```

Comment:
It'd be better to use `outline-color: transparent` to ensure high contrasts mode —which usually drops shadows— can still show the outline properly.

Reply:
```suggestion
    outline-color: transparent;
```

Reply:
Up on this, I really think that'd be a quick accessibility win without any drawbacks 👌